### PR TITLE
Fix/ci

### PR DESCRIPTION
**Pull request by Guillaumebeuzeboc**

### CODE REVIEW

File: .github/workflows/snap.yaml [hunk 1]
```diff
@@ -14,66 +14,10 @@ on:
...
-        SNAPCRAFT_STORE_CREDENTIALS: ${{ secrets.STORE_LOGIN }}
-      with:
-        snap: ${{ env.SNAPFILE }}
-        release: beta # TODO replace by a stable release after proper testing
+  snap:
+    uses: canonical/robotics-actions-workflows/.github/workflows/snap.yaml@main
+    secrets:
+      snapstore-login: ${{ secrets.STORE_LOGIN }}
+    with:
+      runs-on: '["ubuntu-latest", ["self-hosted", "linux", "ARM64", "large", "noble"]]'
```
Comment (artivis) [inline .github/workflows/snap.yaml:22]: Does it really have to be `large`?

File: .github/workflows/keep-alive.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (artivis) [inline .github/workflows/keep-alive.yaml:1]: Do we really need a scheduled workflow (and thus this keep-alive?).

Comment (Guillaumebeuzeboc) [reply .github/workflows/keep-alive.yaml:1]: if we don't, the workflow is going to get disabled. And if the snap build breaks (change in snapcraft), we won't be able to know it.

Comment (Guillaumebeuzeboc) [reply .github/workflows/keep-alive.yaml]: removed the keep alive flow and froze the snapcraft version

**Review by artivis — APPROVED** [review]

**Pull request merged**